JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.html

```diff
@@ -0,0 +1,215 @@
+<div>
+  <dx-resizable
+    class="resizable-container"
+    [minWidth]="500"
+    [minHeight]="150"
+    [maxHeight]="370"
+    handles="right"
+  >
+    <dx-toolbar [multiline]="multiline">
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButton"
+        [options]="undoButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="never"
+        widget="dxButton"
+        [options]="redoButtonOptions"
+      ></dxi-item>
+
+      <dxi-item
+        cssClass="menu-separator-container"
+        location="before"
+        locateInMenu="auto"
+        menuItemTemplate="menuSeparatorTemplate"
+      >
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxDropDownButton"
+        [options]="fontSizeOptions"
+      ></dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxDropDownButton"
+        [style.line-height]="lineHeight"
+        [options]="lineHeightOptions"
+      ></dxi-item>
+
+      <dxi-item
+        cssClass="menu-separator-container"
+        location="before"
+        locateInMenu="auto"
+        menuItemTemplate="menuSeparatorTemplate"
+      >
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
+        locateInMenu="auto"
+        widget="dxSelectBox"
+        [options]="fontSelectorOptions"
+      ></dxi-item>
+
+      <dxi-item
+        cssClass="menu-separator-container"
+        location="before"
+        locateInMenu="auto"
+        menuItemTemplate="menuSeparatorTemplate"
+      >
+        <div *dxTemplate>
+          <div class="separator"></div>
+        </div>
+      </dxi-item>
+
+      <dxi-item
+        location="before"
```

Comment:
Seems that you and Anton decide to avoid using options and use the templates in all approaches? Is not right?

Reply:
Yes